Tests/ApolloCodegenTests/ApolloCodegenConfigurationCodableTests.swift

```diff
@@ -32,24 +32,24 @@ class ApolloCodegenConfigurationCodableTests: XCTestCase {
         output: .init(
           schemaTypes: .init(
             path: "/output/path",
-            moduleType: .swiftPackageManager
+            moduleType: .embeddedInTarget(name: "SomeTarget")
```

Comment:
These changes are to a value other than the default so the test ensures the value is correctly decoded.